README.md

```diff
@@ -59,14 +72,18 @@ helm template randomidhere -f captain.yaml --dependency-update --namespace=glueo
 helm template randomidhere -f captain.yaml --dependency-update --namespace=glueops-core ./admiral/glueops-platform | kubectl -n glueops-core apply -f -
 ```
 
-**_Note: you run the command twice because the CRD's get installed on the very first deployment._**
+**_Notes:_** 
+**_It can take up to 10 minutes for the services on kubernetes to come up.  Login to ArgoCD with `admin` credentials to monitor the status of the deployment. See `Cheat Sheet` below for details about logging in if you need them._**
+**_You run the command twice because the CRD's get installed on the very first deployment._**
+**_Your random id can't start or end with a number._**
```

Comment:
we should probably remove this and maybe just hardcode`captain` instead of telling people to specify something?